app/models/services/locators/doi_paper_locator.rb

```diff
@@ -6,30 +6,32 @@ def column
   def find_paper
     return super if super
 
+    paper = Paper.new
+
+    crossref = Crossref.new
+    if result = crossref.get_doi_metadata(self.locator_id)
```

Comment:
Just a small note, I've been making it clear that a variable has been assigned in the condition by putting the conditional assignment in parentheses. So it would look like `(result = crossref.get_doi_metadata(self.locator_id))`. Not a big deal, but a stylistic preference.